system_tests/caff_node_test.go

```diff
@@ -23,23 +23,21 @@ func createCaffNode(ctx context.Context, t *testing.T, existing *NodeBuilder) (*
 	nodeConfig.Sequencer = false
 	nodeConfig.Dangerous.NoSequencerCoordinator = true
 	execConfig.Sequencer.Enable = false
-	execConfig.Sequencer.EnableCaffNode = true
 	execConfig.ForwardingTarget = existing.l2StackConfig.IPCPath
 	execConfig.SecondaryForwardingTarget = []string{}
-	execConfig.Sequencer.CaffNodeConfig.Namespace = builder.chainConfig.ChainID.Uint64()
-	execConfig.Sequencer.CaffNodeConfig.NextHotshotBlock = 1
-	execConfig.Sequencer.CaffNodeConfig.ParentChainNodeUrl = "http://0.0.0.0:8545"
```

Comment:
No need for the ParentChainNodeUrl anymore? Is it the url of `nodeConfig`?

Reply:
since we've enabled the `parentChainReader`, so a l1 reader is created before the creation of the caff node. We would just use it instead of creating another l1 client.